docs/rhel-installation-guide.md

```diff
@@ -0,0 +1,70 @@
+# Installing Intel® Clear Containers 3.0 on RHEL
```

Comment:
Change heading to infinitive for SEO.
Install Intel® Clear Containers 3.0 on RHEL